bigbluebutton-html5/imports/api/meetings/server/modifiers/removeMeetingFromCollection.js

```diff
@@ -2,9 +2,10 @@ import { clearUsersCollection } from '/imports/api/users/server/modifiers/clearU
 import { clearChatCollection } from '/imports/api/chat/server/modifiers/clearChatCollection';
 import { clearShapesCollection } from '/imports/api/shapes/server/modifiers/clearShapesCollection';
 import { clearSlidesCollection } from '/imports/api/slides/server/modifiers/clearSlidesCollection';
-import { clearPresentationsCollection } from '/imports/api/presentations/server/modifiers/clearPresentationsCollection';
-import {clearMeetingsCollection} from '/imports/api/meetings/server/modifiers/clearMeetingsCollection';
-import { clearPollCollection } from '/imports/api/polls/server/modifiers/clearPollCollection';
+import { clearPresentationsCollection }
+  from '../../../presentations/server/modifiers/clearPresentationsCollection';
+import {clearMeetingsCollection} from '../../../meetings/server/modifiers/clearMeetingsCollection';
+import { clearPollCollection } from '../../../polls/server/modifiers/clearPollCollection';
```

Comment:
I didn't know how else to fix this particular warning.. length of each line is supposed to be no more than 100 characters. It's either a new line or a relative path.
Switching to relative paths was enough to cut the length for some places, but for a few I had to go to a new line.
If you are fine with the new line - I can change relative paths to that way.
Any ideas / suggestions?